### DISCUSSION TITLE
is a bug about the coordinate axis in mtex-6.0.beta2.zip ?

### DISCUSSION THREAD

**Discussion by shaochuyou** [mtex-toolbox/mtex#2084] [Ask Anything]
Hello, everyone
Here are two questions:
1.when For a magnesium sample, the index for pole plane is  (-2111),(-211-1) ,(-2110),and why not is (0001),(10-10),(-2110)?
2.which import wizard by mtex-6.0.beta2.zip(in mtex-5.10-2 everything is ok), the index for pole plane is that always the x is east and y is north always,No matter which one is chosen in Mtex plotting convention?
Look forward to your reply,
Thanks
![Pole figure](https://github.com/mtex-toolbox/mtex/assets/84962655/406e7b8e-ad2e-4eb2-9354-c39771a057c6)

**Comment by kilir** [comment]
Hi,
with respect to 2) please have a look here https://mtex-toolbox.github.io/changelog#1 and also consider using the 6.0beta3 version which has less bugs than the beta2.
Wrt 1) I am not really sure where you get the Miller indices from? Usually you'll specify those somewhere. If you specify (0001),(10-10),(-2110) but in return get (-2111),(-211-1) ,(-2110), please share the lines you of code you used.

Cheers,

Rüdiger

**Reply by shaochuyou** [reply]
Thank you very much for solving my confusion in question 1, and forquestion 2 I found it by using the command 'plot' in the 'import_wizard'.

**Comment by shaochuyou** [comment]
Hello, everyone
Here is another questions:
1.How can I make the red 'x' and 'y' symbols not appear below the plane for {(0001),(10-10),(-2110)} in pole figure?
%% pole figure
h = Miller({0,0,0,1},{1,0,-1,0},{2,-1,-1,0},ebsd('M').CS);
plotPDF(ebsd('Magnesium').orientations,m,'contourf')
% add one colorbar for all sub figures
setColorRange('equal') % set equal color range for all subplots
% add individual colorbars for all sub figures
mtexColorbar('multiple')
![pole figure](https://github.com/mtex-toolbox/mtex/assets/84962655/f0243ded-66c7-48ed-9e83-e85bf8720f97)

**Comment by lfgmorales** [comment]
Hi

1) I think you will need to explain better the first question

2) there are some important changes in MTEX 6 about reference frames, please look here

https://mtex-toolbox.github.io/changelog#1

cheers

Luiz



Luiz F. G. Morales
ETH Zürich
Structural Geology and Tectonics Group
Scientific Center for Optical and Electron Microscopy (ScopeM)
HPM C 57.3
Otto-Stern-Weg 3
8093 Zurich, Switzerland
Phone +41 44 633 37 46
Email: ***@***.******@***.***>



On 6 Mar 2024, at 03:15, shaochuyou ***@***.******@***.***>> wrote:



Hello, everyone
Here are two questions:
1.when For a magnesium sample, the index for pole plane is (-2111),(-211-1) ,(-2110),and why not is (0001),(10-10),(-2110)?
2.which import wizard by mtex-6.0.beta2.zip(in mtex-5.10-2 everything is ok), the index for pole plane is that always the x is east and y is north always,No matter which one is chosen in Mtex plotting convention?
Look forward to your reply,
Thanks
Pole.figure.jpg (view on web)<https://github.com/mtex-toolbox/mtex/assets/84962655/406e7b8e-ad2e-4eb2-9354-c39771a057c6>

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/2084>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN3LJ2DLMJURDINO3J3YWZ32XAVCNFSM6AAAAABEIGUMSOVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGMZDQNBUGU>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/2084", "url": "https://github.com/mtex-toolbox/mtex/discussions/2084", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Comment by lfgmorales** [comment]
Hi

Sorry, Rüdiger have replied your message

for the x-y

setMTEXpref('pfAnnotations',@(varargin) 1);


on the beginning of your script

ccheers

Luiz





Luiz F. G. Morales
ETH Zürich
Structural Geology and Tectonics Group
Scientific Center for Optical and Electron Microscopy (ScopeM)
HPM C 57.3
Otto-Stern-Weg 3
8093 Zurich, Switzerland
Phone +41 44 633 37 46
Email: ***@***.******@***.***>



On 6 Mar 2024, at 10:59, shaochuyou ***@***.******@***.***>> wrote:



Hello, everyone
Here is another questions:
1.How can I make the red 'x' and 'y' symbols not appear below the plane for {(0001),(10-10),(-2110)} in pole figure?
%% pole figure
h = Miller({0,0,0,1},{1,0,-1,0},{2,-1,-1,0},ebsd('M').CS);
plotPDF(ebsd('Magnesium').orientations,m,'contourf')
% add one colorbar for all sub figures
setColorRange('equal') % set equal color range for all subplots
% add individual colorbars for all sub figures
mtexColorbar('multiple')
pole.figure.jpg (view on web)<https://github.com/mtex-toolbox/mtex/assets/84962655/f0243ded-66c7-48ed-9e83-e85bf8720f97>

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/2084#discussioncomment-8691500>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5INYI6UGBLSNJSPDFKDDYW3SJPAVCNFSM6AAAAABEIGUMSOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DMOJRGUYDA>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/2084#discussioncomment-8691500", "url": "https://github.com/mtex-toolbox/mtex/discussions/2084#discussioncomment-8691500", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by shaochuyou** [reply]
Thank you very much my friend, I am a beginner, I am trying to learn the mtex,  in fact, you help me solve the problem that puzzled me for a long time.
